fastlane/Fastfile

```diff
@@ -427,6 +427,11 @@ platform :ios do
   lane :finalize_release do |options|
     UI.user_error!('To finalize a hotfix, please use the finalize_hotfix_release lane instead') if ios_current_branch_is_hotfix
 
+    # Temporarily: we should remove after moving to Xcode 16 in definitive
+    if !UI.confirm('Please ensure Xcode image is set to xcode-15.4 before moving forward, if it is, type y. If not, type n.')
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Automattic/pocket-casts-ios/blob/1708aea88dff68644a6433dbb9216ad26a6736b7/fastlane/Fastfile#L431"></span>Style/NegatedIf: Favor <code>unless</code> over <code>if</code> for negative conditions.</td>
    </tr>
  </tbody>
</table>

Reply:
@danielebogo is that bot friend of yours? 🤣